import traceback as tb
import sys

# def print_stack_trace():
#     # doesn't actually cause an exception
#     try:
#         raise Exception()
#     except Exception:
#         # tb.print_tb(sys.exc_info()[2], limit=20)
#         tb.print_exc(limit=20)
